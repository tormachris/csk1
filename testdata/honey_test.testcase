VerboseMode
XMLPrepare
<!DOCTYPE map
[
<!ELEMENT map>
<!ELEMENT tile(id, type, connectsto)>
<!ELEMENT id (1)
<!ELEMENT type (0)
<!ELEMENT connectsto (2)
<!ELEMENT tile(id, type, connectsto)>
<!ELEMENT id (2)
<!ELEMENT type (0)
<!ELEMENT connectsto (1)
<!ELEMENT tile(id, type, connectsto)>
<!ELEMENT id (3)
<!ELEMENT type (0)
<!ELEMENT connectsto (2)
<!ELEMENT tile(id, type, connectsto)>
<!ELEMENT id (4)
<!ELEMENT type (0)
<!ELEMENT connectsto (3)
<!ELEMENT tile(id, type, connectsto)>
<!ELEMENT id (5)
<!ELEMENT type (0)
<!ELEMENT connectsto (4)
<!ELEMENT thing(type, tileid, force)>
<!ELEMENT type (worker)
<!ELEMENT tileid (1)
<!ELEMENT force (3)
<!ELEMENT thing(type, tileid, force)>
<!ELEMENT type (crate)
<!ELEMENT tileid (2)
<!ELEMENT force (0)
<!ELEMENT thing(type, tileid, force)>
<!ELEMENT type (crate)
<!ELEMENT tileid (3)
<!ELEMENT force (0)
<!ELEMENT thing(type, tileid, force)>
<!ELEMENT type (crate)
<!ELEMENT tileid (4)
<!ELEMENT force (0)
<!ELEMENT frictionmoifier (tileid, type)>
<!ELEMENT tileid (1)
<!ELEMENT type (honey)
<!ELEMENT frictionmoifier (tileid, type)>
<!ELEMENT tileid (2)
<!ELEMENT type (honey)
<!ELEMENT frictionmoifier (tileid, type)>
<!ELEMENT tileid (3)
<!ELEMENT type (honey)
<!ELEMENT frictionmoifier (tileid, type)>
<!ELEMENT tileid (4)
<!ELEMENT type (honey)
<!ELEMENT frictionmoifier (tileid, type)>
<!ELEMENT tileid (5)
<!ELEMENT type (honey)
]>
XMLOver
connectTiles 2 3 NORTH
connectTiles 3 4 NORTH
connectTiles 4 5 NORTH
moveWorker 0 NORTH
