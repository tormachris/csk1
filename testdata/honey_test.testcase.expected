Worker 0 : fmput
false
